src/Chat/Bot/Cipher.hs

```diff
@@ -0,0 +1,46 @@
+module Chat.Bot.Cipher where
+
+import Chat.Data
+import Data.List
+
+
+-- LEVEL: Easy?
```

Comment:
Yeah, I'm on the fence whether to call this `Easy`, or `Medium`. Not that we really have a measure of what that means with `guess` as the only real alternative. Happy to leave it, or change. We'll almost certainly be tweaking the ratings and bots as well.